tests/ApcuCounterTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Tests;
+
+use Yiisoft\Yii\RateLimiter\Counter;
+use Yiisoft\Yii\RateLimiter\Exception\OutOfMaxAttemptsException;
+use Yiisoft\Yii\RateLimiter\Storage\ApcuStorage;
+use Yiisoft\Yii\RateLimiter\Storage\StorageInterface;
+use Yiisoft\Yii\RateLimiter\Tests\FakeApcuStorage;
+
+final class ApcuCounterTest extends BaseCounterTest
+{
+    protected function createStorage(): StorageInterface
+    {
+        return new ApcuStorage();
+    }
+
+    protected function clearStorage(): bool
+    {
+        return apcu_clear_cache();
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->clearStorage();
+    }
+
+    /**
+     * 
+     * Testing that in concurrent scenarios, when dirty reads occur,
+     * the current limiter still performs as expected By 'ApcuStorage'.
+     * 
+     */
+    public function testConcurrentHitsWithDirtyReading(): void
+    {
+        $timer = new FrozenTimeTimer();
+        $storage = new FakeApcuStorage(5);
+        $limit_hits = 10;
```

Comment:
```suggestion
        $limitHits = 10;
```

And other places also. Use camelCase.